src/main/java/org/candlepin/resource/ActivationKeyResource.java

```diff
@@ -283,8 +282,11 @@ public Iterable<ContentOverrideDTO> listActivationKeyContentOverrides(
 
         ActivationKey key = this.fetchActivationKey(activationKeyId);
 
-        CandlepinQuery<ActivationKeyContentOverride> query = this.contentOverrideCurator.getList(key);
-        return this.translator.translateQuery(query, ContentOverrideDTO.class);
+        List<ActivationKeyContentOverride> overrides = this.contentOverrideCurator.getList(key);
+        return overrides.stream().map(this.translator.getStreamMapper(
```

Comment:
Let's drop the `.map` call down a line so the line breaks occur more naturally on individual operations.

(note this applies several times throughout this PR)